certified-connectors/Talkdesk/apiDefinition.swagger.json

```diff
@@ -22,7 +22,7 @@
     "application/json"
   ],
   "paths": {
-    "/webhooks/schema/{listID}": {
+    "/webhooks/schemas/{listID}": {
```

Comment:
@StefanCRT22 The path template has been changed for the operation. can you please confirm that no existing flows and users will be affected by this change?
Is it possible to add it as new operation?

Reply:
@Amjed-Ayoub No existing flows or users will be affected. What's the best option for this change? To create a new action and deprecate this one or just update the existing one?

Reply:
Yes please deprecate the current action and add the new one.
You can also add a note for users in the old operation description, and tell them to use the new one.
Thank you so much.

Reply:
I didn't added the comment because the user will not see this action. Will be used in background. Thank you for your help.